src/main/java/edu/ie3/datamodel/io/processor/EntityProcessor.java

```diff
@@ -258,9 +259,15 @@ private String processMethodResult(Object methodReturnObject, Method method, Str
       case "HpTypeInput":
         resultStringBuilder.append(((UniqueEntity) methodReturnObject).getUuid());
         break;
-      case "Optional": // todo needs to be removed asap as this is very dangerous, but necessary as
-        // long as #75 is not addressed
-        resultStringBuilder.append(((Optional<String>) methodReturnObject).orElse(""));
+      case "EvCharacteristicInput":
+      case "OlmCharacteristicInput":
+      case "WecCharacteristicInput":
+      case "CosPhiFixed":
+      case "CosPhiP":
+      case "QV":
+      case "ReactivePowerCharacteristic":
+      case "CharacteristicInput":
+        resultStringBuilder.append(((CharacteristicInput) methodReturnObject).deSerialize());
```

Comment:
Dunno if this makes sense/works, but it turns out raw unparametrized complains 😅 

```suggestion
        resultStringBuilder.append(((CharacteristicInput<?,?>) methodReturnObject).deSerialize());
```